src/test/unit/controls/Bubble/Bubble/BubbleLoad-spec.js

```diff
@@ -40,6 +42,14 @@ describe("Bubble - Load", () => {
     afterEach(() => {
         document.body.innerHTML = "";
     });
+    beforeAll(() => {
+        // to supress warnings
+        consolewarn = console.warn;
+        console.warn = () => {};
+    });
+    afterAll(() => {
+        console.warn = consolewarn;
+    });
```

Comment:
May i know what is purpose of these warn in before all and after all?

Reply:
This is so there are no warnings printed when running unit tests. The before block temporarily replaces the `console.warn` method and the after block restores it.